src/components/Viewers/TextDisplayer.jsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Typography, Divider } from "@mui/material";
+import { object, bool, func } from "prop-types";
+
+const TextDisplayer = ({ item, onDoubleClick, displayContent }) => {
+  // const partialContent = `${item.content.substring(0, 100)}...`; // Adjust the number based on your needs
+
+  return (
+    <Box sx={{ cursor: "pointer" }} onDoubleClick={onDoubleClick}>
+      {displayContent && <div dangerouslySetInnerHTML={{ __html: item.content_html }}></div>}
```

Comment:
dangerouslySetInnerHtml can pose security risk , please  sanitizing the html , using  dompuirfy library or other  if any